docs/source/developers/guide/step_by_step/pr_and_github.rst

```diff
@@ -127,6 +127,49 @@ If all is set, you can make the pull request!
       and other additional information :ref:`pull_request_and_review`
       section.
 
+Continuous Integration (CI)
+---------------------------
+
+Continuous integration (CI) is an automated way to assert the code
+change made by a specific pull request before it gets merged or
+integrated into the main repository of the project.
+
+Once the pull request is created, the CI will trigger checks on the
+code. Depending on what part of the code was changed (documentation,
+C++ or other languages for example) the CI will choose the corresponding
```

Comment:
May be pedantic, but I might rephrase this a little; the CI doesn't choose - we have configured the CI to run the relevant checks.